data/map systems.txt

```diff
@@ -37239,7 +37239,8 @@ system Sol
 	arrival 4300
 	departure 4189
 	habitable 1496
-	belt 4189 100
+	belt 4189 15
```

Comment:
Doesn't this make the inner asteroid belt very insubstantial?

Reply:
Not at all.  Because the inner belt has a much smaller diameter, the asteroids get clustered much closer together.

![Screenshot_20250201_103718](https://github.com/user-attachments/assets/3cf846d0-c569-4283-a551-bbf2c2acdd78)

Reply:
![Screenshot_20250201_103851](https://github.com/user-attachments/assets/cbcbc3ef-9133-43bf-a669-c2a74b817853)